### PR TITLE
OCD-3941: update test to add new report job in the list of expected jobs for ONC Staff users

### PR DESCRIPTION
**Pull request by ashwinimore**

### CODE REVIEW

File: e2e/pages/administration/jobs/scheduled.readonly.spec.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ describe('the Scheduled Jobs page', () => {
         'All Broken Surveillance Rules Report',
         'Cures Statistics Email',
         'Developer Access Report',
+        'Developer Attestations Check-in Report Email',
```
Comment (andlar) [inline e2e/pages/administration/jobs/scheduled.readonly.spec.js:33]: This was a job that changed an existing report, right? So should one be removed too? Or something else?

**Review by andlar — DISMISSED** [review]

File: e2e/pages/listing/listing.po.js [hunk 2]
```diff
@@ -60,6 +60,7 @@ class ListingPage {
   listingHistoryModalRows() {
     return $(this.elements.listingHistoryTitle)
       .parentElement()
+      .$('div.MuiDialogContent-root.MuiDialogContent-dividers')
```
Comment (andlar) [inline e2e/pages/listing/listing.po.js:63]: Why is this here?

I generally try to avoid locating based on MUI stuff, because we have no control over the class names, and I'd expect any MUI version upgrade to break a lot of things

Comment (ashwinimore) [reply e2e/pages/listing/listing.po.js:63]: without that the test fails to locate the table element in product history modal. The test has been failing for a long time. If you strongly recommend not using it, then I can look for other way to locate it. The table does not have unique id defined.

Comment (ashwinimore) [reply e2e/pages/listing/listing.po.js:63]: or I can revert the change for now and maybe rewrite the test in next PR...open to suggestions

Comment (andlar) [reply e2e/pages/listing/listing.po.js:63]: I don't like this, but if you say it works then I'll call it a win. If the structure changes or we change the content that shows up in this thing at some point, maybe we can revisit, but for now, and the sake of time, It's okay

**Review by andlar — APPROVED** [review]

**Review by athomas-git — APPROVED** [review]
